docs/growth.md

```diff
@@ -1,15 +1,15 @@
-# Growth and Challenge
+# Growth
 
 Growth in our careers is directly proportional to the scale of problems
 we can effectively solve. To grow, we must advocate for ever-larger problems
 and continually learn how to approach and solve them.
 
 Stratasan has a high level of commitment to support our employees' ongoing
 education and career growth. However, actually making use of this is our own
-indvidual responsibility. This may include spending personal time on learning
```

Comment:
So you're saying I misspelled individual differently in the same document. Sounds about right :/